vwifi.c

```diff
@@ -50,6 +57,10 @@ struct owl_wiphy_priv_context {
 struct owl_ndev_priv_context {
     struct owl_context *owl;
     struct wireless_dev wdev;
+    struct net_device *ndev;
+    struct net_device_stats stats;
+    struct list_head rx_queue;
+    struct list_head list;
```

Comment:
The name `list` was misleading. What is your intention?

Reply:
`owl_context` has two net device private data, I use `struct list_head netintf_list` to maintain.

Reply:
Can you add comments to annotate their purposes/intention?

Reply:
Sure, no problem